crates/cadenza-markdown/src/generated/snapshots/cadenza_markdown__generated__test_data__code_block_params__code_block_params_ast.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: crates/cadenza-markdown/src/generated/test_data.rs
+expression: "# Interactive Example\n\nThis example shows code with parameters.\n\n```cadenza editable hidden\nlet setup = initialize()\n```\n\nThe code above is hidden and editable.\n\n```javascript\nconsole.log(\"Hello\");\n```\n\nDifferent languages are supported!\n"
+---
+[
+    [#, "Interactive Example"],
+    "This example shows code with parameters.",
```

Comment:
Standalone strings without calling a macro is going to be difficult to handle. With that we'd have to have a custom evaluator just for MD which I'd really like to avoid.